src/lint/linter/UberCheckstyleLinter.php

```diff
@@ -212,7 +212,7 @@ private function getCommand() {
     $this->checkConfiguration();
     $futures = array();
     // Call checkstyle in batches
-    $chunks = array_chunk($paths, 100);
+    $chunks = array_chunk($paths, $this->maxFiles);
```

Comment:
I believe this caused an outage last time when maxFiles were not configured properly. Can we validate that issue is fixed?